tests/terraform-eks/create/aws-eks.tf

```diff
@@ -0,0 +1,33 @@
+module "eks" {
+  source       = "terraform-aws-modules/eks/aws"
+  cluster_name =  var.eks_cluster_name
+  subnets      = module.vpc.private_subnets
+
+  tags = {
+    Environment = var.environment
+  }
+
+  vpc_id = module.vpc.vpc_id
+
+  worker_groups = [
```

Comment:
what are worker groups? Why do we need two? Why not 1 or 3 or 4?

Reply:
Workers are kubernetes nodes. Worker groups are required to support auto-scaling option. They are the EC2 instances.

Currently we set only variables, as auto-scaling needs more options, and its not currently implemented.
This will be implemented and tested soon: https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/autoscaling.md